### DISCUSSION TITLE
Why is Helix messing with my cursor height? Could it not do that, please?

### DISCUSSION THREAD

**Discussion by effinsky** [helix-editor/helix#6664] [General]
This is zsh in tmux: 
<img width="371" alt="Screenshot 2023-04-09 at 10 06 12" src="https://user-images.githubusercontent.com/54535543/230761688-e36aa136-c203-4ff8-a88a-eeeb298764f7.png">

And this is hx, same terminal instance, same line height:
<img width="784" alt="Screenshot 2023-04-09 at 10 06 41" src="https://user-images.githubusercontent.com/54535543/230761695-b87ef4ec-5fd5-4f6f-bd2e-995bd38140bd.png">

For comparison, this is the same file in nvim:
<img width="404" alt="Screenshot 2023-04-09 at 10 09 38" src="https://user-images.githubusercontent.com/54535543/230761747-93b266b1-4ce1-4de7-972e-e341cfc63e1e.png">

What gives? Is there a way to keep the cursor short and not get stretched vertically? Like maybe it's overriding terminal settings?

**Comment by CptPotato** [comment]
Helix doesn't override any terminal settings. I assume the difference you see is that the first screenshot is using the actual terminal cursor while helix colors the cell's background instead.

To my knowledge there is no way to support multiple cursors in a terminal, so helix must use the cell's background for now.

**Comment by effinsky** [comment]
very interesting to know helix is not actually using the terminal cursor due to this limitation. thanks for the info. I wonder, though, if there could be an improvement to how much it uses of the cell's background. does it need to be the whole thing?
